open-metadata-implementation/access-services/subject-area/subject-area-api/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/properties/relationships/HasA.java

```diff
@@ -62,72 +65,69 @@
     private static final java.util.Set<String> ATTRIBUTE_NAMES_SET = new HashSet<>(Arrays.asList(ATTRIBUTE_NAMES_SET_VALUES));
```

Comment:
And it.

Reply:
I assume this refers to unused code o to be removed in another issue